sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/unit/tests/inputs/DBInputWritable.java

```diff
@@ -0,0 +1,107 @@
+package org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.inputs;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.apache.beam.sdk.io.hadoop.inputformat.integration.tests.HIFIOWithPostgresIT;
+import org.apache.hadoop.io.Writable;
+import org.apache.hadoop.mapreduce.lib.db.DBWritable;
+
+/**
+ * To access the data from RDBMS using {@link DBInputFormat} you have to create a class to define
+ * the data which you are going to read. {@link DBInputWritable} is a class to read data from
+ * Postgres DB in test {@link HIFIOWithPostgresIT}. {@link DBInputWritable} holds id and name of the
+ * scientist.
+ */
+public class DBInputWritable implements Writable, DBWritable {
+  public String id;
+  public String name;
+
+  /* 
+   * Empty constuctor is required for encoding and decoding values in Beam.
```

Comment:
is it required for encoding/decoding in beam, or for Writables? That's two different things.